src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px $catalog-horizontal-padding;
+  margin: 0 auto;
+  width: getCatalogWidth(
```

Comment:
better not change the width of the main tag, you can use one more wrapper inside the main and move catalog class on it
It will fix test